[FW][FIX] stock_currency_valuation: Fix _get_currency_price_unit when sto…

fw-bot-adhoc opened the pull request:
…ck move is created from purchase order

Forward-Port-Of: ingadhoc/stock#724

maq-adhoc committed:
[FIX] stock_currency_valuation: Fix _get_currency_price_unit when stock move is created from purchase order

X-original-commit: e2ad5322c2d67dee05d58479148a9b32fbd538bb

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/stock/pull/725).

fw-bot-adhoc commented:
@maq-adhoc @rov-adhoc cherrypicking of pull request ingadhoc/stock#724 failed.

stdout:
```
CONFLICT (modify/delete): stock_currency_valuation/models/stock_move.py deleted in HEAD and modified in e0d7ac03 ([FIX] stock_currency_valuation: Fix _get_currency_price_unit when stock move is created from purchase order). Version e0d7ac03 ([FIX] stock_currency_valuation: Fix _get_currency_price_unit when stock move is created from purchase order) of stock_currency_valuation/models/stock_move.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/models/stock_landed_cost.py deleted in HEAD and modified in e0d7ac03 ([FIX] stock_currency_valuation: Fix _get_currency_price_unit when stock move is created from purchase order). Version e0d7ac03 ([FIX] stock_currency_valuation: Fix _get_currency_price_unit when stock move is created from purchase order) of stock_currency_valuation/models/stock_landed_cost.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/__manifest__.py deleted in HEAD and modified in e0d7ac03 ([FIX] stock_currency_valuation: Fix _get_currency_price_unit when stock move is created from purchase order). Version e0d7ac03 ([FIX] stock_currency_valuation: Fix _get_currency_price_unit when stock move is created from purchase order) of stock_currency_valuation/__manifest__.py left in tree.

```

stderr:
```
20:56:55.242446 git.c:444               trace: built-in: git cherry-pick e0d7ac037bb15e9e208d8d2c2032480aa639548a
error: could not apply e0d7ac03... [FIX] stock_currency_valuation: Fix _get_currency_price_unit when stock move is created from purchase order
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port

rov-adhoc commented:
@maq-adhoc Filo este tienen que cerrarse y pasar directo a 18

maq-adhoc commented:
Cierro
